composer.json

```diff
@@ -8,8 +8,8 @@
   "description": "Bonus points plugin for Sylius.",
   "license": "MIT",
   "require": {
-    "php": "^7.3",
-    "sylius/sylius": "^1.9.0"
+    "php": "^7.4 || ^8.0",
+    "sylius/sylius": "^1.10.0"
```

Comment:
I saw lot of plugins which are compatible with 1.10 | 1.9 | 1.8. In this plugin not possible? @tgrochowski

Reply:
You're right - those 3 supported versions were added to composer.json - all GH actions tests went ok